docs/develop/01_sdk/03_chain_setup/01_standalone_setup.md

```diff
@@ -62,3 +62,52 @@ const devFaucet = Crypto.makeKeypairFromUri(faucetSeed)
 ```
 
 With the new `devFaucet`, you can transfer funds to other accounts and test all the KILT features that require tx fee payment.
+
+## Standalone vs. Parachain (Peregrine/Spiritnet)
+
+The standalone chain is close in functionality to Kilt parachains but there are a few fundamental differences between them.
+
+<!-- ### Blocktime
+
+Block time is actually the same, but this might change in the future. -->
```

Comment:
blocktime is actually different. It's 12 seconds for parachains and 1 seconds for our standalone chain. We also gave the standalone chain the name Kestrel 🐦.

Reply:
@weichweich This is commented out anyway, I assume for this reason. Will note to look at this seperately.